shared/js/default_activity_helper.js

```diff
@@ -0,0 +1,151 @@
+/* global SettingsHelper */
```

Comment:
Maybe I already asked before but I can't remember: why does this file need to be in `shared`?